### PR TITLE
Allen/bity

### PR DESCRIPTION
**Pull request by ahujsak**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/constants/plugins/buySellPlugins.js [hunk 1]
```diff
@@ -65,6 +65,14 @@ export const allPlugins: Array<BuySellPlugin> = [
     supportEmail: 'support@sendwyre.com',
     permissions: ['camera']
   },
+  {
+    pluginId: 'co.edgesecure.bity',
+    uri: hostedUri + 'co.edgesecure.bity/index.html',
+    name: 'Bity',
+    subtitle: 'ACH bank transfer\nBTC\nFee: 1% / Settlement: 1-5 days\nBuy Limit: $2500 / Unlimited Sell',
+    imageUrl: 'https://edge.app/wp-content/uploads/2019/09/logoBity.png',
+    supportEmail: 'support@sendwyre.com'
```
Comment (swansontec) [inline src/constants/plugins/buySellPlugins.js]: Wrong email address.

File: src/constants/plugins/buySellPlugins.js [hunk 2]
```diff
@@ -65,6 +65,14 @@ export const allPlugins: Array<BuySellPlugin> = [
     supportEmail: 'support@sendwyre.com',
     permissions: ['camera']
   },
+  {
+    pluginId: 'co.edgesecure.bity',
```
Comment (swansontec) [inline src/constants/plugins/buySellPlugins.js]: This pluginId sucks.

File: src/modules/UI/scenes/Plugins/EdgeProvider.js [hunk 3]
```diff
@@ -339,19 +339,17 @@ export class EdgeProvider extends Bridgeable {
...
-    // this is about bit id signatures.
-    const obj = {
-      publicKey: 'string',
-      // Hex encoded signature
-      signedMessage: 'string'
+    try {
+      const signedMessage = await coreWallet.otherMethods.signMessageBase64(message, guiWallet.receiveAddress.publicAddress)
+      return signedMessage
+    } catch (e) {
+      console.log('EP: E', e)
```
Comment (swansontec) [inline src/modules/UI/scenes/Plugins/EdgeProvider.js]: This should be:

```js
catch (e) {
  showError(e)
  throw e // <- This way Bity knows it failed? Unless they know some other way?
}
```

File: copy-bity.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# Usage: copy-core [path-to-core-ui]
```
Comment (swansontec) [inline copy-bity.sh]: Wrong comment?

File: src/modules/UI/scenes/Plugins/EdgeProvider.js (hunk 3 above)
Comment (ahujsak) [reply src/modules/UI/scenes/Plugins/EdgeProvider.js]: thanks, however Bity handles calling the `showError`. I figure it is better to let the individual plugins decide what to do with an error, and then if they want to use ours they can all `EdgeProvider` back and we can then show it.

File: src/constants/plugins/buySellPlugins.js (hunk 2 above)
Comment (ahujsak) [reply src/constants/plugins/buySellPlugins.js]: co.edgesecure.bity mirrors what we did with wyre and simplex. I am open to suggestions.

File: copy-bity.sh (hunk 4 above)
Comment (ahujsak) [reply copy-bity.sh]: fixed

**Pull request merged**